composeApp/src/commonMain/kotlin/io/github/drp08/studypal/navigation/BottomNavItem.kt

```diff
@@ -4,24 +4,25 @@ import androidx.compose.material.icons.Icons
 import androidx.compose.material.icons.filled.DateRange
 import androidx.compose.material.icons.filled.Home
 import androidx.compose.material.icons.filled.Person
-import androidx.compose.runtime.Composable
+import androidx.compose.material.icons.filled.Settings
 import androidx.compose.ui.graphics.vector.ImageVector
 import cafe.adriel.voyager.core.screen.Screen
 import io.github.drp08.studypal.screens.CalendarScreen
 import io.github.drp08.studypal.screens.HomeScreen
-import io.github.drp08.studypal.screens.PomodoroScreen
 import io.github.drp08.studypal.screens.ProfileScreen
+import io.github.drp08.studypal.screens.RegistrationScreen
 
 sealed class BottomNavItem(
     val label: String,
     val icon: ImageVector,
     val screen: Screen,
 ) {
     companion object {
-        fun values() = listOf(Home, Calendar, Profile)
+        fun values() = listOf(Home, Calendar, Profile, Register)
```

Comment:
Okay for now, maybe.. But remember we need to make this only on first install of the application.